lib/src/posthog_flutter_io.dart

```diff
@@ -257,6 +257,21 @@ class PosthogFlutterIO extends PosthogFlutterPlatformInterface {
     }
   }
 
+  @override
+  Future<bool> isOptOut() async {
+    if (!isSupportedPlatform()) {
+      return true;
+    }
+
+    try {
+      final result = await _methodChannel.invokeMethod('isOptOut');
+      return result as bool? ?? true;
```

Comment:
I debated whether falling back to `true` was the right behavior here (and elsewhere!)